src/node.h

```diff
@@ -375,8 +375,6 @@ NODE_DEPRECATED("Use WinapiErrnoException(isolate, ...)",
 })
 #endif
 
-const char *signo_string(int errorno);
```

Comment:
Not sure what you mean… e.g. addons do use the symbols exported from the `node` binary, yes.